### DISCUSSION TITLE
Adding a Custom Vehicle to Docker Carla

### DISCUSSION THREAD

**Discussion by hashim19** [carla-simulator/carla#5589] [Q&A]
Hi All, 

I need help with adding my new custom vehicle to the docker version of carla. I have already imported the vehicle into carla built from source using [this](https://carla.readthedocs.io/en/0.9.13/tuto_A_add_vehicle/) tutorial. What I wanna do now is to import this vehicle to my docker image of the carla. Can anyone list out the step on how I can achieve that? 

Thanks in advance.

**Comment by LawranceXcl** [comment]
build a docker file, like as follow:
![image](https://user-images.githubusercontent.com/47016472/181140344-d0334672-d43d-4b05-ab77-715c53d69da7.png)
then build yourself carla docker image
